smart-contract-verifier/src/http_server/handlers/verification/solidity/contract_verifier.rs

```diff
@@ -79,9 +79,31 @@ async fn compile_and_verify(
     let compiler_output = compilers
         .compile(&input.compiler_version, &input.compiler_input)
         .await?;
+    let compiler_output_modified = {
```

Comment:
just checking: we can't compile only the original, check somehow easily for matches, and if nothing matched only then compile additional stuff? As I understood, we just threw away the old algo, so we need modified outputs for new?

Reply:
That's an interesting idea, thanks. Probably we can do that. However, I believe it would be better to make that at a separate PR and maybe after we understand that verification has started to take considerable amount of time we want to reduce